src/main/resources/openapi/hearings.yaml

```diff
@@ -244,3 +244,15 @@ components:
       enum:
         - "AUTHORISATION_100"
       x-enum-varnames: [ USER_NOT_AUTHORISED_FOR_COURTHOUSE ]
+
+    HearingsErrorCode:
+      type: string
+      enum:
+        - "100"
```

Comment:
Would need an op name prefixing them with the api name i.e. something like HEARINGS_100 to ensure system wide uniqueness in the error code

Reply:
This would apply across all of the yaml files.

Reply:
Agree, the intent of the `type` field in the JSON response is to uniquely identify the error across the application.

It's hard to ensure a purely numeric value will be unique, because the enum definitons are spread across multiple openapi specs. Whereas adding a prefix (aligned with the feature name, e.g. HEARINGS) effectively namespaces the numeric values, minimising the chances of clashes with values used in other specs.

Reply:
This was always my concern that the consumer would lose info. Happy to go back and refactor to include the prefix in the open api yaml.